pineapple-pictures.json

```diff
@@ -0,0 +1,70 @@
+{
+    "app-id": "net.blumia.pineapple-pictures",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-22.08",
+    "sdk": "org.kde.Sdk",
+    "command": "ppic",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
```

Comment:
this is a very broad filesystem permission.

Reply:
Since this application requires access to the folder that the current opened file is located at, to provide the gallery navigation feature, I guess this one is required? Gwenview also have this permission listed btw.